.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I think it's better to omit the branch name here, so that a pull request will always trigger actions, regardless of what branch it is destined for.